neo/Persistence/Snapshot.cs

```diff
@@ -179,7 +179,7 @@ public ECPoint[] GetValidators()
             return _validators;
         }
 
-        public IEnumerable<ECPoint> GetValidators(IEnumerable<Transaction> others)
+        virtual internal IEnumerable<ECPoint> GetValidators(IEnumerable<Transaction> others)
```

Comment:
The virtual was needed to mock it.... and since it was not used anywhere else,I made it internal to protect a little bit more.. but it could be virtual public too.

Reply:
https://github.com/neo-project/neo/blob/3a06f9c0efcc3cc267d380bb945fe7474b1f2243/neo/Persistence/Snapshot.cs#L21

I think you should mock `Validators`, it's `abstract`.

Reply:
Good idea, I'll try.